driver-sync/src/test/functional/com/mongodb/client/OcspTest.java

```diff
@@ -0,0 +1,32 @@
+package com.mongodb.client;
+
+import com.mongodb.MongoException;
+import com.mongodb.MongoTimeoutException;
+import org.bson.BsonDocument;
+import org.bson.BsonInt32;
+import org.junit.Test;
+
+import static com.mongodb.ClusterFixture.getOcspShouldSucceed;
+
+import static org.junit.Assert.fail;
+
+public class OcspTest {
+    @Test
+    public void testTLS() {
+        String options = "tls=true";
+        try {
+            connect(options);
```

Comment:
Nit: doesn't seem too much point in passing in a constant like this.  Just add it directly to the URL

Reply:
Done